tokio/src/runtime/metrics/runtime.rs

```diff
@@ -242,6 +242,59 @@ impl RuntimeMetrics {
                     .load(Relaxed)
             }
 
+            /// Returns the total number of times the given worker thread has parked
+            /// and unparked.
+            ///
+            /// The worker park/unpark count starts at zero when the runtime is created
+            /// and increases by one each time the worker parks the thread waiting for
+            /// new inbound events to process. This usually means the worker has processed
+            /// all pending work and is currently idle. When new work becomes available,
+            /// the worker is unparked and the park/unpark count is again increased by one.
+            ///
+            /// The counter is monotonically increasing. It is never decremented or
+            /// reset to zero. An odd count means that the worker is currently parked;
+            /// an even count means that the worker is active.
```

Comment:
The paragraph starting with "The counter is monotonically increasing." is duplicated on most methods here, so putting the information about odd/even in this paragraph makes it more likely that it will be missed. I think it would be good to move the odd/even note somewhere else in this doc-comment.

Reply:
I moved the note into its own paragraph.